src/gc/sample/gcenv.h

```diff
@@ -105,7 +105,7 @@ class Thread
 
     bool PreemptiveGCDisabled()
     {
-        return !!m_fPreemptiveGCDisabled;
+        return m_fPreemptiveGCDisabled ? true : false;
```

Comment:
This is not an improvement. `!!` is a fine way to normalize a value to bool. We use it in many places across the codebase.